iceberg/openhouse/internalcatalog/src/main/java/com/linkedin/openhouse/internal/catalog/repository/HouseTableRepositoryImpl.java

```diff
@@ -39,12 +39,15 @@
 public class HouseTableRepositoryImpl implements HouseTableRepository {
 
   /**
-   * The request timeout is decided based on retry template logic and server side gateway timeout of
-   * 60 sec. The retry template has retry max attempt of 3 with 2 secs delay (with delay multiplier
-   * as attempt increases) between each retry. So the overall retry process should complete within
-   * 60 sec.
+   * The read request timeout is decided based on retry template logic and server side gateway
+   * timeout of 180 sec. The retry template has retry max attempt of 3 with 2 secs delay (with delay
+   * multiplier as attempt increases) between each retry. So the overall retry process should
+   * complete within 180 sec.
    */
-  private static final int REQUEST_TIMEOUT_SECONDS = 17;
+  private static final int READ_REQUEST_TIMEOUT_SECONDS = 45;
```

Comment:
45 might be too much for just reading considering we have retries. Lets start with 30.